auth/auth-server/src/server/handle_external_match/mod.rs

```diff
@@ -414,7 +438,10 @@ impl Server {
 
         // Compute refund amount
         let refund_amount = self
-            .get_refund_amount(&external_match_resp.match_bundle.match_result, refund_native_eth)
+            .compute_refund_amount_for_order(
+                &external_match_resp.match_bundle.match_result,
```

Comment:
this is an expected compiler error at the moment, I will update this code path in a follow-up PR